kernels/volk/volk_32fc_s32f_power_spectrum_32f.h

```diff
@@ -128,7 +128,10 @@ volk_32fc_s32f_power_spectrum_32f_a_sse3(float* logPowerOutput, const lv_32fc_t*
     const float real = *inputPtr++ * iNormalizationFactor;
     const float imag = *inputPtr++ * iNormalizationFactor;
 
-    *destPtr = 10.0*log10f(((real * real) + (imag * imag)) + 1e-20);
+    // *destPtr = 10.0*log10f(((real * real) + (imag * imag)) + 1e-20);
+    // For performance rewrite log10 to use log2
```

Comment:
I don't think these comments are needed. (Likewise for the other places they appear below.)